### PR TITLE
Implement Login Command Phase 5

### PR DESCRIPTION
**Pull request by jjlee050**
- Fix mask and unmask bug for textfield.
- Allow receptionist to login.
- Combine doctor and receptionist class to one Staff class
- Refactor isSameAsHashedPassword to checkPassword

### CODE REVIEW

File: src/test/java/seedu/clinicio/model/password/PasswordTest.java [hunk 1]
```diff
@@ -83,19 +83,19 @@ public void isValidHashedPassword() {
...
 
         //invalid password
-        assertFalse(Password.isSameAsHashPassword("peter13", HashUtil.hashToString(password)));
-        assertFalse(Password.isSameAsHashPassword("peter13", HashUtil.hashToString(password)));
-        assertFalse(Password.isSameAsHashPassword("peter12", " ")); //Only spaces password hash string
-        assertFalse(Password.isSameAsHashPassword("", " ")); //Empty string and only spaces password hash string
+        assertFalse(Password.verifyPassword("peter13", HashUtil.hashToString(password)));
+        assertFalse(Password.verifyPassword("peter13", HashUtil.hashToString(password)));
+        assertFalse(Password.verifyPassword("peter12", " ")); //Only spaces password hash string
+        assertFalse(Password.verifyPassword("", " ")); //Empty string and only spaces password hash string
```
Comment (iamjackslayer) [inline src/test/java/seedu/clinicio/model/password/PasswordTest.java]: maybe can add one more test case: "\\\s" -> should be invalid password

Comment (jjlee050) [reply src/test/java/seedu/clinicio/model/password/PasswordTest.java]: ok. hang on

**Comment by jjlee050** [comment]
@gingivitiss Can check your appointment if there is anything affect in ur main and test classes. Coz the refactor may have touch on those classes.

**Comment by gingivitiss** [comment]
@jjlee050 i already checked through and fixed them

**Pull request merged**